tests/smoke_tests/docker/docker_utils.py

```diff
@@ -27,10 +27,20 @@ def get_container_name() -> str:
     return container_name
 
 
+def get_host_port() -> int:
+    """Get the host port that will be listened by the test container."""
+    container_name = get_container_name()
```

Comment:
BTW, is container_name collision possible? e.g. some containers are orphaned in previous test